src/System.Globalization.Extensions/tests/IdnMapping/Data/ConformanceIdnaUnicodeTestResult.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Globalization.Tests
+{
+    public sealed class ConformanceIdnaUnicodeTestResult : ConformanceIdnaTestResult
+    {
+        public bool ValidDomainName { get; private set; }
+
+        public ConformanceIdnaUnicodeTestResult(string entry, string fallbackValue, bool validDomainName = true)
+            : base(entry, fallbackValue)
+        {
+            ValidDomainName = validDomainName;
+        }
+    }
+}
```

Comment:
Yeah, I thought about just introducing a new property to the existing class but I decided against it as IMO subclassing makes more sense here. Of course you could argue that for test cases this isn't that relevant... If okay I would like to stick with the current solution.